modules/SpyUtils.js

```diff
@@ -42,6 +46,17 @@ export const createSpy = (fn, restore = noop) => {
     return returnValue
   }
 
+  if (supportsConfigurableFnLength) {
+    spy = Object.defineProperty(spyLogic, 'length',
+      { value: fn.length, writable: false, enumerable: false, configurable: true })
+  } else {
+    spy = new Function('spy', `return function(${ // eslint-disable-line no-new-func
+      Array(...Array(fn.length)).map((_, i) => Array(i + 2).join('_')).join(',')
```

Comment:
You could simplify this with `[...Array(fn.length)].map((_, i) => '$' + i).join(',')`